distribution/changelog.txt

```diff
@@ -59,6 +59,7 @@
 - Improved: [#6022] Allow up to 128 ride objects to be selected in track designer.
 - Improved: [#12917] Changed peep movement so that they stay more spread out over the full width of single tile paths.
 - Improved: [#13386] A GUI error message is now displayed if the language files are missing.
+- Improved: [#14193] Various openrct2.d.ts improvements.
```

Comment:
I'd rather have multiple lines with the same PR number than having a vague changelog entry.

Reply:
I grouped the changes into three entries.